snuba/query/exceptions.py

```diff
@@ -50,3 +50,8 @@ def __init__(
     @classmethod
     def from_args(cls, exception_type: str, message: str) -> "QueryPlanException":
         return cls(exception_type=exception_type, message=message)
+
+
+class MaxRowsEnforcerException(SerializableException):
```

Comment:
This exception  name does not tell me anything except the component that threw it. Why would this exception be thrown?

Reply:
I think you could name it `TooManyDeleteRows` or something